MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,342 @@
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+    
+    using Syncfusion.Maui.Core.Hosting;
+
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Creating data object
+
+The `SfDataForm` is a data edit control so, create a data object with details to create a data form based on your business requirement.
+
+Here, the data object named **ContactsInfo** created with some properties.
+
+{% tabs %}
+{% highlight C# %}
+
+
+    public class ContactsInfo
+    {
+        public string FirstName { get; set; }
+    
+        public string MiddleName { get; set; }
+    
+        public string LastName { get; set; }
+    
+        public string ContactNumber { get; set; }
+    
+        public string Email { get; set; }
+   
+        public string Address { get; set; }
+
+        public DateTime? BirthDate { get; set; }
+    
+        public string GroupName { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Initialize the data object in view model class in order to bind in 'DataObject' property of 'SfDataForm'.
+
+{% tabs %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public ContactsInfo ContactsInfo {get; set;}
+        
+        public DataFormViewModel()
+        {
+            this.ContactsInfo = new ContactsInfo();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Set data object to data form
+
+By default, data form auto generate the editors based on the primitive data type in the 'DataObject' property. Please refer the following code to set the 'DataObject' property.
+
+{% tabs %}
+{% highlight XAML %}
+
+    <ContentPage 
+        . . .
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:DataFormViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding ContactsInfo}"/>    
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+    dataForm.DataObject = new ContactsInfo();
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Defining editors
+
+The data form control automatically generates `SfDataForm.Items` (which has UI settings of data field) based on the data type in the `SfDataForm.DataObject` property. The `SfDataForm.Items` encapsulates the layout of label and editor setting for the data field appearing in the dataform. 
```

Comment:
The data form control automatically generates the `SfDataForm.Items` (which has UI settings of the data field) based on the data type in the `SfDataForm.DataObject` property. The `SfDataForm.Items` summarizes the layout of the label and editor setting for the data field appearing in the dataform.

Reply:
content updated